app/experimenter/nimbus-ui/src/components/AppLayoutWithExperiment/index.tsx

```diff
@@ -44,11 +56,40 @@ const AppLayoutWithExperiment = ({
   const {
     experiment,
     notFound,
-    loading,
+    loading: experimentLoading,
     startPolling,
     stopPolling,
     review,
   } = useExperiment(slug);
+  // We won't know if an analysis lookup is required until the initial experiment query
+  // is complete and we inspect its status. To prevent content from flashing on the screen
+  // between the experiment and analysis requests, assume we need to wait for analysis
+  // until we can prove otherwise.
+  const [analysisRequired, setAnalysisRequired] = useState<boolean>(true);
+  const [analysisFetched, setAnalysisFetched] = useState<boolean>(false);
+
+  const {
+    execute: fetchAnalysis,
+    result: analysis,
+    loading: analysisLoading,
+  } = useAnalysis();
+
+  useEffect(() => {
+    if (
+      !analysisFetched &&
+      !experimentLoading &&
+      experimentLocked(experiment?.status)
```

Comment:
It looks like we have a nearly identical `experimentLocked` in `AppLayoutWithSidebar`.

Was going to suggest DRYing that up, but I think in _this_ instance, we only want to check if the experiment is live or complete, not "accepted" since results will never be available in that state, in which case the name `experimentLocked` may also not make sense (I like it for accepted/live/complete statuses though).

![image](https://user-images.githubusercontent.com/13018240/101089775-3e747080-357b-11eb-8a61-520cf9241649.png)

Reply:
Related: https://jira.mozilla.com/browse/EXP-653

I've removed the Accepted criteria!